<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QtDemo</class>
 <widget class="QMainWindow" name="QtDemo">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>861</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>双目相机标定程序v1.0</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QPushButton" name="btnSelectFolder">
        <property name="text">
         <string>选择图片文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectSaveFile">
        <property name="text">
         <string>选择保存文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSingleCalibrate">
        <property name="text">
         <string>单目标定</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="1" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_3">
      <item>
       <widget class="QPushButton" name="btnSelectLeftFolder">
        <property name="text">
         <string>选择左相机图片文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectLeftCalibFile">
        <property name="text">
         <string>选择左相机标定文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectRightFolder">
        <property name="text">
         <string>选择右相机图片文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectRightCalibFile">
        <property name="text">
         <string>选择右相机标定文件</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnSelectStereoSaveFolder">
        <property name="text">
         <string>选择双目标定保存文件夹</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="btnStereoCalibrate">
        <property name="text">
         <string>双目标定</string>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="2" column="0">
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <widget class="QLabel" name="labelCornerSize">
        <property name="text">
         <string>角点尺寸 (宽x高):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinCornerWidth">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>11</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinCornerHeight">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>20</number>
        </property>
        <property name="value">
         <number>8</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QLabel" name="labelSquareSize">
        <property name="text">
         <string>棋盘格子大小 (宽x高):</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinSquareWidth">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QSpinBox" name="spinSquareHeight">
        <property name="minimum">
         <number>1</number>
        </property>
        <property name="maximum">
         <number>100</number>
        </property>
        <property name="value">
         <number>25</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item row="3" column="0">
     <layout class="QVBoxLayout" name="verticalLayout">
      <item>
       <widget class="QScrollArea" name="scrollArea">
        <property name="widgetResizable">
         <bool>true</bool>
        </property>
        <widget class="QWidget" name="scrollAreaWidgetContents">
         <property name="geometry">
          <rect>
           <x>0</x>
           <y>0</y>
           <width>839</width>
           <height>175</height>
          </rect>
         </property>
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>175</height>
          </size>
         </property>
        </widget>
       </widget>
      </item>
      <item>
       <widget class="QTextEdit" name="textEdit">
        <property name="readOnly">
         <bool>true</bool>
        </property>
        <property name="tabStopDistance">
         <double>80.000000000000000</double>
        </property>
       </widget>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
